main.go

```diff
@@ -61,6 +67,20 @@ func main() {
 	flag.BoolVar(&debug, constants.ArgNameDebug, false, "Enable verbose output for debugging")
 	flag.Parse()
 
+	if *help {
+		flag.PrintDefaults()
+		return
+	}
+
+	if *versionFlag {
+		fmt.Printf(`%s:
+				go version  : %s
+				go compiler : %s
+				platform    : %s/%s
+			   `, os.Args[0], runtime.Version(), runtime.Compiler, runtime.GOOS, runtime.GOARCH)
```

Comment:
which one is the binary version?

Reply:
I actually removed the binary version stuff so the commit message is a bit wrong, have to add in the VERSION stuff like the runner to do that, and maintain a VERSION.txt file